RNCallKeep.podspec

```diff
@@ -13,6 +13,6 @@ Pod::Spec.new do |s|
   s.requires_arc        = true
   s.platform            = :ios, "8.0"
   s.source_files        = "ios/RNCallKeep/*.{h,m}"
-  s.dependency 'React/Core'
+  s.dependency 'React-Core'
```

Comment:
Merging this will mean 0.60 is the *minimum* version of React Native supported, right? Unless we can come up with a way to set this based on the react-native dependency's version, this will have to wait. There's too many parts of the matrix to require Call Keep users to move all the way to 0.60 now.

Reply:
yeah, minimum rn `60+`

Reply:
I think this needs to hang out a while then, maybe in a beta tagged branch @sboily? The dependency ecosystem is too complex to push to 0.60 as a minimum at this point.

Reply:
Yes we can't do that fo the moment, react-native 0.60 is too young and we have a major risk to broke many applications. I'm open to have branch or other solution for people want to have 0.60+.

Reply:
I tried with `s.dependency 'React'`. It works just fine with RN 0.60.5. So I think it can work for RN <0.60 too.

Reply:
@kylekurz It should work with 0.5x. Because `React` is a base library for all.
```
pod 'React', :path => '../node_modules/react-native'
```

Reply:
Please, merge with `pod 'React', :path => '../node_modules/react-native'`